code/dijkstra-algorithm/src/main/java/ru/unn/agile/DijkstraAlgorithm/Model/DijkstraGraph.java

```diff
@@ -0,0 +1,166 @@
+package ru.unn.agile.DijkstraAlgorithm.Model;
```

Comment:
Please correct the package naming, package name should be in a low case.

Reply:
fixed

Reply:
fixed